src/index.rs

```diff
@@ -887,6 +891,36 @@ impl Index {
     Ok(entries)
   }
 
+  pub(crate) fn runes_paginated(
+    &self,
+    page_size: u64,
+    page_index: u64,
+  ) -> Result<(Vec<RuneInfo>, bool)> {
+    let rtx = self.database.begin_read()?;
+    let number_to_rune_info = rtx.open_table(NUMBER_TO_RUNE_INFO)?;
```

Comment:
Do this with the `RUNE_ID_TO_RUNE_ENTRY` table.

Reply:
if use `RUNE_ID_TO_RUNE_ENTRY` I can't use `range`:
```rust
let runes_info = number_to_rune_info
      .range(end..=start)?
      .rev()
      .map(|result| result.map(|(_number, info)| RuneInfo::load(info.value())))
      .collect::<Result<Vec<RuneInfo>, StorageError>>()?;
```
do you have any suggestions?

Reply:
I think we can move `RUNE_ID_TO_RUNE_ENTRY` and `RUNE_TO_RUNE_ID` 
into `RUNE_ID_TO_NUMBER` and `RUNE_TO_NUMBER`